WcaOnRails/app/models/competition.rb

```diff
@@ -66,10 +65,10 @@ class Competition < ActiveRecord::Base
     self.persisted? || (name.length <= MAX_NAME_LENGTH && name =~ VALID_NAME_RE)
   end
   validates :name, length: { maximum: MAX_NAME_LENGTH },
-                   format: { with: VALID_NAME_RE, message: INVALID_NAME_MESSAGE }
+                   format: { with: VALID_NAME_RE, message: I18n.t('competitions.errors.invalid_name_message') }
```

Comment:
I'm not sure: I think you're concerned that we would have to restart the server if we change the translation, am I right?
It seems to be executed each time we trigger a validation; I tried changing the validation error message without restarting the server and I did see the change :)